e2e/libs/volume/crd.py

```diff
@@ -224,6 +224,16 @@ def wait_for_volume_state(self, volume_name, desired_state):
             time.sleep(self.retry_interval)
         assert volume["status"]["state"] == desired_state
 
+    def wait_for_volume_keep_at_state(self, volume_name, desired_state):
```

Comment:
`at_state` is used here, but `in_state` is used elsewhere. Could we make it consistent?